src/components/organisms/AppRouter/AdminSubRouter.tsx

```diff
@@ -49,6 +51,12 @@ export const AdminSubRouter: React.FC = () => (
       </Provided>
     </Route>
 
+    <Route path={[ADMIN_IA_WORLD_TOOLS]}>
```

Comment:
Other routes and with `_PARAM_URL` to denote their structure/purpose

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2747/commits/73bc0106ad8f80a3da7b69b082166b60d78fc4e8